include/nbl/video/utilities/CAssetConverter.h

```diff
@@ -440,6 +440,7 @@ class CAssetConverter : public core::IReferenceCounted
 				{
 					public:
 						virtual const patch_t<asset::ICPUSampler>* operator()(const lookup_t<asset::ICPUSampler>&) const = 0;
+						virtual const patch_t<asset::IShader>* operator()(const lookup_t<asset::IShader>&) const = 0;
```

Comment:
you don't have a patch, so don't make the method pure or virtual at all, see the methods below with comment
```cpp
// certain items are not patchable, so there's no `patch_t` with non zero size
```